tests/test_executors.py

```diff
@@ -186,6 +187,8 @@ class TaskMockCrash:
 
     def runQuantum(self):
         _LOG.debug("TaskMockCrash.runQuantum")
+        # Disable fault handler to suppress long scary traceback.
+        faulthandler.disable()
```

Comment:
I'm not familiar with `faulthandler`, but this looks like a global setting. Don't you need to re-enable it when the test is complete (not sure how to do this if the task object is responsible for the setting, though).

Reply:
This is indeed a global setting for the whole process, but in this case it's fine because the process that runs the task is killed immediately on the next line. This task is only ever used in a forked execution (otherwise signal would kill the whole test) so it does not affect `faulthandler` in the parent process.

Reply:
Could you add a comment to that effect? It's not at all obvious in context that the task must only be run in an isolated environment.

Reply:
Sorry, I pushed "Merge" already. I'll try to remember adding that on some other ticket.